pkg/analysis/helpers/markers/analyzer.go

```diff
@@ -12,6 +12,8 @@ import (
 	"golang.org/x/tools/go/ast/inspector"
 )
 
+const UnnamedExpression = ""
```

Comment:
Add Godoc to this please to explain when and why this is used

Reply:
https://github.com/JoelSpeed/kal/pull/7/commits/8156a607d1efbaa3cb3702e4c5b7e712925d482f